lab2/css/style.mainnav.scss

```diff
@@ -0,0 +1,36 @@
+$mainnewsbg: #f7f9fa;
+$font-ptsans: 'PT Sans';
+$font-ptserif: 'PT Serif';
+$computerwidth: 940px;
+$mobilewidth: 640px;
```

Comment:
I've forgotten to mention variables. So could you please move them into a separate file as well to avoid code duplication?